west.yml

```diff
@@ -40,7 +40,7 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: 0023611bbfabd0698a38a12e6f30d7a2a9108702
+      revision: pull/148/head
```

Comment:
This can now be changed to a SHA, and after this is merged we need a patch to the changelog.